frontend/src/utils/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from "./validation"
+export * from "./showToastOnError"
+export * from "./date"
```

Comment:
I think this re-exporting like this generally not a great pattern since it can have a negative effect on tree shaking. How do you feel about just importing from the modules themselves?

Reply:
You can still import them by their own modules. I added this to reduce number of refactoring needed since a lot of our codes were using single import.

Reply:
Yeah and thats part of the reason why I'm not crazy about this. It introduces having multiple ways of importing so you end up with less uniform code. The other thing I would wonder about is if this has a negative affect on tree shaking.

Reply:
That's a valid concern. If I were to refactor other files to update the import, I might end up having to fix other pages that have not yet been screened by codacy - which will make this PR large and quite long to fix other non-related errors to this PR. 

How about we address that in another PR? - like update util imports in `EditUser.tsx`, `login.tsx`, ..etc.